modules/openthread/Kconfig.features

```diff
@@ -15,16 +15,16 @@ config OPENTHREAD_THREAD_VERSION_1_2
 	bool "Version 1.2"
 config OPENTHREAD_THREAD_VERSION_1_3
 	bool "Version 1.3"
-config OPENTHREAD_THREAD_VERSION_1_3_1
-	bool "Version 1.3.1"
+config OPENTHREAD_THREAD_VERSION_1_4
+	bool "Version 1.4"
 endchoice # OPENTHREAD_STACK_VERSION
 
 config OPENTHREAD_THREAD_VERSION
 	string
 	default "1.1" if OPENTHREAD_THREAD_VERSION_1_1
 	default "1.2" if OPENTHREAD_THREAD_VERSION_1_2
 	default "1.3" if OPENTHREAD_THREAD_VERSION_1_3
-	default "1.3.1" if OPENTHREAD_THREAD_VERSION_1_3_1
+	default "1.4" if OPENTHREAD_THREAD_VERSION_1_4
```

Comment:
Version 1_3_1 is still available in OT https://github.com/openthread/openthread/blob/main/etc/cmake/options.cmake#L288

So let's not remove that.

Reply:
oh, it must have not been aligned yet. ok, restored it and will remove once OT aligns version